packages/lib/src/interactions/hooks.ts

```diff
@@ -115,66 +115,56 @@ export function useCanvasEvents(callbacks: CanvasEventCallbacks): void {
   const { onPointerDown, onPointerMove, onPointerUp, onWheel } = callbacks;
   const { domElement } = useThree((state) => state.gl);
   const camera = useThree((state) => state.camera);
-  const size = useThree((state) => state.size);
+  const { worldToData, cameraToHtmlMatrixInverse } = useAxisSystemContext();
 
-  const getUnprojectedPoint = useCallback(
-    (evt: PointerEvent | WheelEvent) => {
-      const { offsetX: x, offsetY: y } = evt;
-      const { width, height } = size;
-      const normX = (x - width / 2) / (width / 2);
-      const normY = -(y - height / 2) / (height / 2);
+  const processEvent = useCallback(
+    <T extends PointerEvent | WheelEvent>(sourceEvent: T): CanvasEvent<T> => {
+      const { offsetX, offsetY } = sourceEvent;
 
-      return new Vector3(normX, normY, 0).unproject(camera);
+      const htmlPt = new Vector3(offsetX, offsetY, 0);
+      const cameraPt = htmlPt.clone().applyMatrix4(cameraToHtmlMatrixInverse);
```

Comment:
This replaces the manual html-to-camera transformation that we were doing previously. `cameraToHtmlMatrixInverse` is provided through context.